41_snapshots-btrfs

```diff
@@ -115,6 +115,13 @@ else
     LINUX_ROOT_DEVICE=UUID=${root_uuid}
 fi
 }
+## Detect rootflags
+detect_rootflags()
+{
+    local fstabflags=$(grep -oE '^\s*[^#][[:graph:]]+\s+/\s+btrfs\s+[[:graph:]]+' ${gbgmp}/${snap_dir_name}/etc/fstab \
```

Comment:
`${gbgmp}/${snap_dir_name}/etc/fstab`
needs `" "`  if spaces in their string
`"${gbgmp}/${snap_dir_name}/etc/fstab"`

e.g: `"${gbgmp}/snapshots with spaces/etc/fstab"`
without double quotes, an error will occur.

Reply:
Of course, thanks ! :+1: 

Fixed.